erpnext/crm/report/lead_details/lead_details.py

```diff
@@ -37,10 +47,30 @@ def get_columns():
 			"options": "Territory",
 			"width": 100,
 		},
-		{"label": _("Source"), "fieldname": "utm_source", "fieldtype": "Data", "width": 120},
-		{"label": _("Email"), "fieldname": "email_id", "fieldtype": "Data", "width": 120},
-		{"label": _("Mobile"), "fieldname": "mobile_no", "fieldtype": "Data", "width": 120},
-		{"label": _("Phone"), "fieldname": "phone", "fieldtype": "Data", "width": 120},
+		{
+			"label": _("Source"),
+			"fieldname": "source",
```

Comment:
This fieldname was "utm_source" source before. Please don't make unrelated changes like this.